react/Spinner/Readme.md

```diff
@@ -0,0 +1,49 @@
+Show a spinner when loading something.
+
+### Default 
+
+```
+<Spinner />
+```
+
+### Color
+
+```
+<div>
+  blue: <Spinner color='blue' />
+  grey: <Spinner color='grey' />
+  white: <Spinner color='white' />
+  red: <Spinner color='red' />
+</div>
+```
+
+### Placement
+
+```
+<div>
+<Spinner noMargin={ true }/>
+</div>
+```
+
+### Sizes
+
+```
+<div>
+  tiny: <Spinner size='tiny' /><br/>
+  small: <Spinner size='small' /><br/>
+  medium: <Spinner size='medium' /><br/>
+  large: <Spinner size='large' /><br/>
+  xlarge: <Spinner size='xlarge' /><br/>
+  xxlarge: <Spinner size='xxlarge' />
+</div>
+```
+
+
+### Message
+
+When you use `loadingType`, `<Spinner />` needs to be in an `<I18n />` wrapper as it uses `t` function.
+
+
+```jsx static
+<Spinner loadingType='salut' />
+```
```

Comment:
I don't want `cozy-ui` to be dependent on `<I18n />`.
But maybe, I am alone on this idea.

So please, consider this comment as a non blocking one, we should update later.

By the way, could you find a way to perform a running demo, maybe you can pass through a props like `t={word => word}`.

Reply:
IMHO, this is not a problem of the docs.

Reply:
It allows to display a running demo.